src/frontend/AboutFile.ui

```diff
@@ -6,153 +6,152 @@
    <rect>
```

Comment:
![about_layout_mistakes](https://user-images.githubusercontent.com/9459891/43120682-ea106c10-8f3c-11e8-946d-a8816354272e.png)
That `Ok` button is too much wide :)